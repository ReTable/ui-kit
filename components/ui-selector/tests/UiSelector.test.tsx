import { create } from 'react-test-renderer';
import { describe, expect, it } from 'vitest';

import { UiSelector } from '~';

describe('UiSelector', () => {
  it('works', () => {
    const tree = create(<UiSelector />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
