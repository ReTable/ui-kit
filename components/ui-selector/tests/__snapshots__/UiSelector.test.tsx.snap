// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UiSelector > works 1`] = `
<div
  className="tabula_ui_selector__1r2c28e0"
/>
`;
